documentation/api/query/v4/reports.markdown

```diff
@@ -190,19 +188,17 @@ The `<expanded logs>` object returns all a single log line per data entry as fol
 
 File and line may each be null if the log does not concern a resource.
 
-**Note on fields that allow `NULL` values**
-
-In the resource_event schema above, `containment_path`, `new_value`, `old_value`, `property`, `file`, `line`, `status`, and `message` may all be null.
-
-**Note on querying resource events, metrics, and logs**
+>**Note: Fields that allow `NULL` values**
+>
+>In the resource_event schema above, `containment_path`, `new_value`, `old_value`, >`property`, `file`, `line`, `status`, and `message` may all be null.
 
-The `reports` endpoint does not support querying on the value of `resource_events`, `logs`,
-or `metrics`. For `resource_events` the same information can be accessed by querying the `events` endpoint for events with field `report` equal to a given report's `hash`.
-Making metrics and logs queryable may be the target of future work.
+>**Note: On querying resource events, metrics, and logs**
+>
+>The `reports` endpoint does not support querying on the value of `resource_events`, >`logs`, or `metrics`. For `resource_events`, the same information can be accessed by >querying the `events` endpoint for events with field `report` equal to a given report's >`hash`. Making metrics and logs queryable may be the target of future work.
```

Comment:
not sure how this renders but it seems like the >'s should each have their own line